flang-rt/unittests/CMakeLists.txt

```diff
@@ -6,33 +6,39 @@
 #
 #===------------------------------------------------------------------------===#
 
+# Target that depends on all unittests
+add_custom_target(FlangRTUnitTests)
+set_target_properties(FlangRTUnitTests PROPERTIES FOLDER "Flang-RT/Meta")
+
 # LLVM uses a modified version of GTest that uses LLVMSupport for console
```

Comment:
This comment remains to imply "do not expect to be able to substitute upstream gtest", right?

Which is what I get from reading the new form of the comment, but just checking.

Reply:
Correct. The build target is called `llvm_gtest`, so it will not work with upstream gtest.